sdks/wasm/framework/src/WebAssembly.Bindings/Runtime.cs

```diff
@@ -346,9 +346,13 @@ static string GetCallSignature (IntPtr method_handle)
 					res += "s";
 					break;
 				default:
-					if (t.IsValueType)
-						throw new Exception ("Can't handle VT arguments");
-					res += "o";
+					if (t == typeof(IntPtr)) { 
+ 						res += "i";
```

Comment:
non-commit-blocking question: Is this function hot? It looks like it could potentially get called frequently. Under those circumstances it might be better for this to use a stringbuilder or a fixed-size char[] buffer that gets converted into a string

Reply:
It's call during any kind of interop with Javascript. In the context of this PR, it's also used for callbacks. So yes, optimizations there will make sense. Currently, it does not appear sharply in the profiler, as what's called is generally a lot more costly.

Reply:
After a second thought, it is only be called when registering a method: 

https://github.com/jeromelaban/mono/blob/527abcd11521282476d53f101c3b39d6c46d2ae3/sdks/wasm/binding_support.js#L723